### ISSUE TITLE
For treeGrid there should be a topic for keyBoard navigation

### ISSUE THREAD

**Issue by nrobakova** [IgniteUI/igniteui-docfx#804]
There should be an information for keyBoard navigation:
- Row can be collapsed with Alt+Arrow Left key
- Row can be expanded with Alt+ Arrow Right key

**Comment by gedinakova** [comment]
This should probably be part of a general Keyboard Navigation topic as the only part specific to TreeGrid is the expand/collapse shortcut as described here.
